dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_12__Rename_All_AREA_Type_To_STACKED_AREA.sql

```diff
@@ -0,0 +1,4 @@
+
+-- Rename all AREA type to STACKED_AREA, in the "visualization" table.
+
+UPDATE visualization SET "type"='STACKED_AREA' WHERE trim("type")='AREA';
```

Comment:
Do we need "trim" here? Just wondering since we have an enum for type so there should not be any invalid values in that column.

Reply:
That's right, it was just my sense of avoiding any issues during migration. But of course, it can be removed. Will push soon.

Reply:
Done